### PR TITLE
Metrics Pipeline

### PR DESCRIPTION
**Pull request by rahul-179**

### CODE REVIEW

File: components/job/component.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  - {name: tags,        type: List,     default: '[]',
+     description: 'Optional. List of user-chosen tags to allow Runs to be better identified or grouped.'}
+outputs:
+  - {name: rundetails,      description: 'Details of the dkube run'}
+  - {name: artifact,        description: 'Identifier in Dkube storage where artifact generated by this component are stored.'}
+  - {name: mlpipeline-ui-metadata, description: 'metadata file to display metrics'}
+  - {name: output, description: 'output file from the job'}
+implementation:
+  container:
+    image: ocdr/dkube_pylauncher:2.2
```
Comment (deepio-oc) [inline components/job/component.yaml]: this needs to be 2.3 right ?

File: metrics-pipeline/config.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+code:
+  name: dkube-examples
+  url: https://github.com/rahul-179/dkube-examples.git
```
Comment (deepio-oc) [inline metrics-pipeline/config.yaml]: this needs to change to dkube-examples

File: metrics-pipeline/metrics-pipeline.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import json
```
Comment (deepio-oc) [inline metrics-pipeline/metrics-pipeline.py]: rename 'metrics-pipeline' to just metrics or metrics-visualization  (rename metrics.py if needed)

**Comment by rahul-179** [comment]
Screenshot from 2021-09-22 04-36-14

**Comment by rahul-179** [comment]
Along with the above 2 output , also added table

Screenshot from 2021-09-22 20-32-45

**Comment by deepio-oc** [comment]
@rahul-179 could you explore how to get metrics under metrics section in the pipeline UI?

**Review by deepio-oc — APPROVED** [review]

**Pull request merged**